adapters-api/src/main/java/io/nosqlbench/engine/api/activityimpl/uniform/DryRunOp.java

```diff
@@ -0,0 +1,16 @@
+package io.nosqlbench.engine.api.activityimpl.uniform;
+
+import io.nosqlbench.engine.api.activityimpl.uniform.flowtypes.Op;
+import io.nosqlbench.engine.api.activityimpl.uniform.flowtypes.RunnableOp;
+
+public class DryRunOp implements RunnableOp {
+
+    private final Op op;
+
+    public DryRunOp(Op op) {
+        this.op = op;
+    }
+    @Override
+    public void run() {
```

Comment:
Do we want to log anything at debug level, or would that be a bit verbose for these ops?